apps/ios/Shared/Views/UserSettings/IncognitoHelp.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  IncognitoHelp.swift
+//  SimpleX (iOS)
+//
+//  Created by JRoberts on 22.08.2022.
+//  Copyright © 2022 SimpleX Chat. All rights reserved.
+//
+
+import SwiftUI
+
+struct IncognitoHelp: View {
+    var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text("Incognito mode")
+                .font(.largeTitle)
+                .padding(.vertical)
+            ScrollView {
+                VStack(alignment: .leading) {
+                    Group {
+                        Text("Turning Incognito mode on allows you to connect to new contacts and join groups without sharing your main profile. When incognito mode is enabled, for each new conversation a new random profile is generated and sent to a person or group you're connecting to.")
+                        Text("When accepting contact requests received via a contact address, you can decide to connect using a random profile by enabling Incognito mode, even if you've created the address itself outside of Incognito mode.")
+                        Text("To learn which profile is used in an incognito conversation, open chat information page — for direct contacts this profile is available under \"Your random profile\", and for groups it's your group membership profile.")
+                    }
+                    .padding(.bottom)
+
+                    Group {
+                        Text("Groups")
+                            .font(.title2)
+                            .padding(.top)
+                        Text("To minimize user confusion, we've made several decisions regarding how groups operate in Incognito mode:")
+                        textListItem("•", "When you create a group in Incognito mode or join incognito, the same random profile is shared with all group members;")
+                        textListItem("•", "If your membership in a group is not incognito (i.e. with your main profile), inviting contacts with whom you've connected incognito to this group is prohibited to avoid the possibility of other members sharing or mentioning your main profile to the new member - such contacts will be disabled when inviting group members;")
```

Comment:
```suggestion
                        textListItem("•", "If you joined the group with your main profile, you will not be able to invite the contacts with whom you've shared incognito profile – to avoid sharing your main profile with them.")
```